lib/potassium/assets/app/frontend/utils/query-params.ts

```diff
@@ -0,0 +1,8 @@
+import { decamelizeKeys } from 'humps';
+import { stringify } from 'qs';
+
+export function paramsSerializer(params: object) {
+  const decamelizedParams = decamelizeKeys(params);
+
+  return stringify(decamelizedParams, { arrayFormat: 'repeat' });
```

Comment:
Sorry, didn't see this before. I think it has to the `brackets` here to get the usual `array[]=1&array[]=2` format rails uses

Reply:
Oh, good catch. I though it didn't matter but never tested it 😬